package.json

```diff
@@ -17,14 +17,14 @@
   },
   "main": "index.js",
   "scripts": {
-    "eslint": "eslint client common server test",
-    "jscs": "jscs client common server test",
+    "lint": "eslint client common server test && jscs client common server test",
```

Comment:
To make this work with the coming soon CI support, you should rewrite as `eslint . && jscs .`, then add a .eslintignore and an "excludeFiles" property in your .jscsrc. If your .eslintignore is identical to your .gitignore, then you can do `eslint --ignore-path=.gitignore` (CI will do that, too).